bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/ScriptExecution.java

```diff
@@ -91,6 +91,10 @@ public static Timer createTimer(AbstractInstant instant, Procedure0 closure) {
             dataMap.put("procedure", closure);
             dataMap.put("timer", timer);
             JobDetail job = newJob(TimerExecutionJob.class).withIdentity(jobKey).usingJobData(dataMap).build();
+            if (TimerImpl.scheduler.checkExists(job.getKey())) {
+                TimerImpl.scheduler.deleteJob(job.getKey());
```

Comment:
do we have any certainty that this existing job has also been created by the same code? It must imho be avoided that jobs from someone else get deleted.
I cannot really make much sense from the job key - how does that string look like? It should be something human readable, is this the case here?